unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -15,6 +15,17 @@ import {
   getHeaders,
 } from '../../../scripts/utils.js';
 
+const DOS_SPECIAL_NAMES = new Set([
+  'CON', 'PRN', 'AUX', 'NUL', 'COM0', 'COM1', 'COM2', 'COM3', 'COM4', 'COM5', 'COM6',
+  'COM7', 'COM8', 'COM9', 'LPT0', 'LPT1', 'LPT2', 'LPT3', 'LPT4', 'LPT5', 'LPT6',
+  'LPT7', 'LPT8', 'LPT9'
+]);
+
+// Pre-compile regular expressions to be used in sanitizeFileName
+const INVALID_CHARS_REGEX = /[\x00-\x1F\\/:"*?<>|]/g;
+const ENDING_SPACE_PERIOD_REGEX = /[ .]+$/;   // Handles multiple trailing spaces/periods
+const STARTING_SPACE_PERIOD_REGEX = /^[ .]+/; // Handles multiple leading spaces/periods
```

Comment:
We generally avoid adding comments to the file for performance reasons. Could we please remove them here?